src/screens/Watch/model.js

```diff
@@ -80,6 +80,7 @@ const initState = {
     bulkEditing: false,
     updating: false,
     currCaptionIndex: 0,
+    captionSpeedUp: 0,
```

Comment:
Speed up suggests a rate change. "time offset" would be more accurate

Reply:
ill make this change